com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolume.hlsl

```diff
@@ -190,7 +190,7 @@ bool TryToGetPoolUVW(APVResources apvRes, float3 posWS, float3 normalWS, out flo
 
     // unpack pool idx
     // size is encoded in the upper 4 bits
-    uint   subdiv = (packed_pool_idx >> 28) & 15;
+    subdiv = (packed_pool_idx >> 28) & 15;
```

Comment:
Note (for me, I will fix in a PR after this), this still thinks there are 4 bits for subdiv, while we actually have 7